src/machine/image-dbus.c

```diff
@@ -17,6 +17,7 @@
   along with systemd; If not, see <http://www.gnu.org/licenses/>.
 ***/
 
+#include <sys/file.h>
```

Comment:
LOCK_xyz is a BSD API, not a POSIX one, I figure POSIX doesn't matter then... That said, the Linux man pages suggest including sys/file.h for the BSD locking API, hence this appears OK to me.